Lab 1.ipynb

```diff
@@ -0,0 +1,581 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Recreating first binomial plot\n",
+    "\n",
+    "k = 2\n",
+    "p = .6\n",
+    "\n",
+    "fig, ax = plt.subplots(1, 1)\n",
+    "x = np.arange(0,50, 0.5)\n",
+    "y = stats.binom.pmf(k, x, p, loc = -10)\n",
+    "\n",
+    "plt.tick_params(labelsize = 12)\n",
+    "ax.set_ylabel('Probability', fontsize = 12)\n",
+    "ax.set_xlabel('Value', fontsize = 12)\n",
+    "\n",
+    "ax.plot(x,y, 'bo')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "70599db7-cda3-434b-b0b4-03d292e233e9",
+   "metadata": {},
+   "source": [
+    "Statistical Question: What is the chance that a value of 30 or higher is due to random chance as opposed to a signal? <br>\n",
+    "Method: Sum probabilities of all values above the test value of 30."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "beb80127-c734-43d5-8c41-0285c53f870d",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.021239876109960594\n",
+      "-2.028789333210327\n"
+     ]
+    }
+   ],
+   "source": [
+    "prob = stats.binom.cdf(k, 30, p, loc = -10)\n",
+    "sigma = stats.norm.ppf(prob, loc = 0, scale = 1)\n",
+    "print(prob)\n",
+    "print(sigma)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bd3ca1fa-5f26-4b61-8cfd-4b17d7b0f59c",
+   "metadata": {},
+   "source": [
+    "Probabilities of all observed values of discrete, and the observed values themselves are discrete, so finding the probability above or below a certain point (where the integral from before has now become a sum of everything past the test point) will also be discrete. Hence the sigmas following from these values are discrete- they have to follow whatever probability is found. With continuous distributions you had a histogram which was matched with a continuous function that could have the same probability for a few observed values (because all of these observed values were fit in the same bin if they were close enough). With non-continuous distributions you should know exactly where the value belongs (because there is only certain allowed values), so you should be able to get a better idea of exactly where your value is in terms of its sigma.\n",
```

Comment:
I think you missed a word or something in your first sentence? Could be edited for clarity